lib/mix/tasks/openapi.spec.json.ex

```diff
@@ -20,6 +20,7 @@ defmodule Mix.Tasks.Openapi.Spec.Json do
   * `--vendor-extensions` - Whether to include open_api_spex OpenAPI vendor extensions
     (defaults to true)
 
+  * `--quiet` - Whether to disable output printing
```

Comment:
Please mention the default, `(defaults to false)`.

Reply:
Done!